StartioAdmobMediation/StartioAdmobAdapter.m

```diff
@@ -77,7 +77,7 @@ + (void)setupStartioSDKWithMediationParameters:(StartioAdmobParameters *)mediati
     sdk.appID = mediationParameters.appId;
     sdk.returnAdEnabled = NO;
     sdk.consentDialogEnabled = NO;
```

Comment:
you may remove line 78 and 79 because this is done inside enableMediationModeFor